src/main/io/ledstrip.c

```diff
@@ -277,6 +279,34 @@ STATIC_UNIT_TESTED void updateDimensions(void)
 
 }
 
+enum ledBarIds {
+   LED_BAR_GPS,
+   LED_BAR_BATTERY,
+   LED_BAR_COUNT
+};
+static uint8_t ledBarStates[LED_BAR_COUNT] = {0};
```

Comment:
```suggestion
static uint8_t ledBarLedCount[LED_BAR_COUNT] = {0};
```

Reply:
unresolved?

Reply:
We can refactor later if we want.

Reply:
I think `ledBarStates` is the right label here

Reply:
will approve, maybe wait for @jonas-becker & @ledvinap reply. up to ya'll.

Reply:
I would also keep it as `ledBarStates`. If not, it could also conflict with `ledBarCounters`.